tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/OpenPaasContactsConsumer.java

```diff
@@ -59,6 +59,11 @@
 import reactor.rabbitmq.Receiver;
 import reactor.rabbitmq.Sender;
 
+/**
+ * This class is deprecated and will be removed in a future version.
+ * Use {@link com.linagora.tmail.contact.SabreContactsConsumer} instead.
+ */
+@Deprecated(forRemoval = true)
```

Comment:
I would remove it right away: it do not change interfaces.
WHere sabre is deployed so is openpaas.